ea_apply_normalization_tofile.m

```diff
@@ -58,6 +58,7 @@ function ea_apply_normalization_tofile(options,from,to,directory,useinverse,inte
         ea_fsl_apply_normalization(options,from,to,useinverse,refim,'',interp);
 
     otherwise % SPM part here
+        tempdir = pwd;% ensure using a writable folder (not necessarily tempdir)
```

Comment:
Do you mean that the default tempdir can be not writable in some cases?

Reply:
this line can be dangerous since the original .nii.gz file might be deleted if it is in the current folder.